Library/Homebrew/global.rb

```diff
@@ -43,8 +44,14 @@
 
 # Bintray fallback is here for people auto-updating from a version where
 # HOMEBREW_BOTTLE_DEFAULT_DOMAIN isn't set.
-HOMEBREW_BOTTLE_DEFAULT_DOMAIN = ENV["HOMEBREW_BOTTLE_DEFAULT_DOMAIN"] ||
-                                 "https://homebrew.bintray.com"
+if ENV["HOMEBREW_BOTTLE_DEFAULT_DOMAIN"]
```

Comment:
Sorry, last pedantic thing:

`HOMEBREW_BOTTLE_DEFAULT_DOMAIN = if ENV["HOMEBREW_BOTTLE_DEFAULT_DOMAIN"]` means you can avoid specifying `HOMEBREW_BOTTLE_DEFAULT_DOMAIN =` three times.